### PR TITLE
Improve function documentation

### PR DESCRIPTION
**Pull request by danningyu**

### CODE REVIEW

File: table/fib-strategy-tree.go [hunk 1]
```diff
@@ -61,6 +63,8 @@ func (f *fibStrategyTreeEntry) findExactMatchEntry(name *ndn.Name) *fibStrategyT
 	return nil
 }
 
+// findExactMatchEntry returns the entry corresponding to the longest
```
Comment (zjkmxy) [inline table/fib-strategy-tree.go]: Function name is wrong.

File: table/fib-strategy.go [hunk 2]
```diff
@@ -18,6 +18,8 @@ type FibStrategyEntry interface {
 	GetNextHops() []*FibNextHopEntry
 }
 
+// baseFibStrategyEntry represents an implementation of FibStrategyEntry
```
Comment (zjkmxy) [inline table/fib-strategy.go]: I think it is only a base class, without implementing all functions of FibStrategyEntry

File: table/pit-cs-tree.go [hunk 3]
```diff
@@ -73,8 +73,8 @@ func (e *nameTreePitEntry) PitCs() PitCsTable {
 	return e.pitCsTable
 }
 
-// FindOrInsertPIT inserts an entry in the PIT upon receipt of an Interest. Returns tuple of PIT entry and whether the Nonce is a duplicate.
-// Originally implemented as FindOrInsertPIT()
+// FindOrInsertPIT inserts an entry in the PIT upon receipt of an Interest.
```
Comment (zjkmxy) [inline table/pit-cs-tree.go]: Function name is wrong.

**Review by zjkmxy — COMMENTED** [review]
Just some typo.
FYI: go does not have a maximum line length. The practice of all codebases I maintained (YaNFD, python-ndn, and go-ndn in future) uses **120** characters instead of typical 80. But you don't need to merge existing short lines.

File: table/fib-strategy.go (hunk 2 above)
Comment (danningyu) [reply table/fib-strategy.go]: Updated description

**Review by zjkmxy — APPROVED** [review]
Looks good to me.

**Pull request merged**